packages/core/src/blocks/defaultBlocks.ts

```diff
@@ -30,6 +30,7 @@ import { FileBlock } from "./FileBlockContent/FileBlockContent";
 import { ImageBlock } from "./ImageBlockContent/ImageBlockContent";
```

Comment:
Looking at this, we will probably want to move this file out of the `blocks` directory and rename it to `defaultSchema`. It's fine for now though, just leaving this as a note to sort this out later.